pact/Pact/Core/Gas.hs

```diff
@@ -121,7 +102,9 @@ data ZKGroup
   -- ^ Group one, that is Fq in Pairing
   | ZKG2
   -- ^ Group two, that is, Fq2 Pairing
-  deriving (Show, Generic, NFData)
+  deriving (Show, Generic)
+
+instance NFData ZKGroup
```

Comment:
Is there a reason that all of the other types were not moved to `Pact.Gas.Types`?

Just a nitpick on our current core convention, but I think that our gas types should most likely be in one place, this includes gas args.

Reply:
Thx, done